packages/perseus/src/widgets/lights-puzzle.tsx

```diff
@@ -7,8 +7,36 @@ import _ from "underscore";
 import * as Changeable from "../mixins/changeable";
 import WidgetJsonifyDeprecated from "../mixins/widget-jsonify-deprecated";
 
+import type {PerseusLightsPuzzleWidgetOptions} from "../perseus-types";
 import type {WidgetExports} from "../types";
 
+// Types
```

Comment:
:sad-panda: I've been trying to delete this widget for about 2 years. lol

Reply:
Got the approval to delete the widget now! :D